wp-content/plugins/core/src/Assets/Assets_Subscriber.php

```diff
@@ -19,6 +19,10 @@ public function register(): void {
 			$this->container->get( Admin_Assets_Enqueuer::class )->register();
 		}, 10, 0 );
 
+		add_action( 'enqueue_block_editor_assets', function (): void {
```

Comment:
I think we want to use `enqueue_block_assets` instead based on [this dev note for  Core 6.3](https://make.wordpress.org/core/2023/07/18/miscellaneous-editor-changes-in-wordpress-6-3/#post-editor-iframed).

> It’s also worth noting that enqueue_block_editor_assets should not be used to add stylesheets _for editor content_. Please use enqueue_block_assets instead.